Library/Homebrew/utils.rb

```diff
@@ -770,3 +770,23 @@ def number_readable(number)
   (numstr.size - 3).step(1, -3) { |i| numstr.insert(i, ",") }
   numstr
 end
+
+def truncate_text_to_approximate_size(s, max_bytes, opts = {})
+  front_weight = opts[:front_weight] || 0.5
+  glue = opts[:gloe] || "\n[...snip...]\n"
```

Comment:
Fair enough. Switched this to an inline constant and no externally-exposed options.